pcornet/v3/schema/harvest.csv

```diff
@@ -4,27 +4,27 @@ pcornet,v3,harvest,network_name,string,20,,,
 pcornet,v3,harvest,datamartid,string,10,,,
 pcornet,v3,harvest,datamart_name,string,20,,,
 pcornet,v3,harvest,datamart_platform,string,2,,,
-pcornet,v3,harvest,cdm_version,number,,16,8,
+pcornet,v3,harvest,cdm_version,number,,10,2,
 pcornet,v3,harvest,datamart_claims,string,2,,,
 pcornet,v3,harvest,datamart_ehr,string,2,,,
-pcornet,v3,harvest,birth_date_mgmt,string,,,,
-pcornet,v3,harvest,enr_start_date_mgmt,string,,,,
-pcornet,v3,harvest,enr_end_date_mgmt,string,,,,
-pcornet,v3,harvest,admit_date_mgmt,string,,,,
-pcornet,v3,harvest,discharge_date_mgmt,string,,,,
-pcornet,v3,harvest,px_date_mgmt,string,,,,
-pcornet,v3,harvest,rx_order_date_mgmt,string,,,,
-pcornet,v3,harvest,rx_start_date_mgmt,string,,,,
-pcornet,v3,harvest,rx_end_date_mgmt,string,,,,
-pcornet,v3,harvest,dispense_date_mgmt,string,,,,
-pcornet,v3,harvest,lab_order_date_mgmt,string,,,,
-pcornet,v3,harvest,specimen_date_mgmt,string,,,,
-pcornet,v3,harvest,result_date_mgmt,string,,,,
-pcornet,v3,harvest,measure_date_mgmt,string,,,,
-pcornet,v3,harvest,onset_date_mgmt,string,,,,
-pcornet,v3,harvest,report_date_mgmt,string,,,,
-pcornet,v3,harvest,resolve_date_mgmt,string,,,,
-pcornet,v3,harvest,pro_date_mgmt,string,,,,
+pcornet,v3,harvest,birth_date_mgmt,string,2,,,
+pcornet,v3,harvest,enr_start_date_mgmt,string,2,,
```

Comment:
Here's the error. I unintentionally removed some commas from this line and those following. Fixing now.